tests/integration-tests/resources/file-cache-storage-cfn.yaml

```diff
@@ -0,0 +1,261 @@
+
+Description: This template to creates FSX Filecache storage that can be attached to a cluster for Integration Test.
+
+Parameters:
+  FileCachePath:
+    Description: Fsx FileCachePath for Data Repository Association.
+    Type: String
+    Default: "/fsx-cache-path/"
+  S3BucketName:
+    Description: S3 Bucket used as Data Repository for FSX File Cache.
+    Type: String
+    Default: ""
+  VpcId:
+    Description: VPC ID to for adding FileCache SecurityGroup
+    Type: AWS::EC2::VPC::Id
+    Default: ""
+  SubnetId:
+    Description: Subnet ID.
+    Type: AWS::EC2::Subnet::Id
+    Default: ""
+
+Resources:
+
+  FsxFileCacheSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Allow TCP Traffic
```

Comment:
I suggest to be more specific in the SG description, e.g. "Allow traffic for FSx File Cache"

Reply:
Got it.